tailwind.config.js

```diff
@@ -1,7 +1,35 @@
 module.exports = {
-  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  content: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
+
   theme: {
     extend: {},
   },
-  plugins: [],
+  plugins: [require('daisyui')],
+  daisyui: {
```

Comment:
do clear is out with TAs/MAs of NeogCamp if were allowed to use any UI plugins , hope that doesnt get marks cut later.